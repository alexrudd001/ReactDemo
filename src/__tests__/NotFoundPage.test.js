import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NotFoundPage from '../components/NotFoundPage';

test('renders Header correctly', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(toJson(wrapper)).toMatchSnapshot();
});