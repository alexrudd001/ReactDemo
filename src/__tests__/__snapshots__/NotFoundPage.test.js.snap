// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header correctly 1`] = `
<div>
  Error : 404. Page not found. 
  <br />
   
  <Link
    to="/"
  >
     Go home
  </Link>
</div>
`;
